sql-plugin-api/src/main/scala/com/nvidia/spark/rapids/ShimLoader.scala

```diff
@@ -53,7 +53,7 @@ import org.apache.spark.util.MutableURLClassLoader
     jar:file:/home/spark/rapids-4-spark_2.12-24.08.0.jar!/spark320/
     Spark 3.1.1 will use
     jar:file:/home/spark/rapids-4-spark_2.12-24.08.0.jar!/spark-shared/
-    jar:file:/home/spark/rapids-4-spark_2.12-24.08.0.jar!/spark311/
+    jar:file:/home/spark/rapids-4-spark_2.12-24.08.0.jar!/spark320/
```

Comment:
Similar comment as above.  Spark 3.2.0 is already discussed above, this is about Spark 3.1.1.  Should change this section to describe what happens for Spark 3.3.0 or something similar.

Reply:
checking